jac-uk/admin
# #2058 Allow blank scenario scores => did not take test

## What's included?

Allows data entry to be completed even if some applications do not have scores entered into the platform. These will be interpreted as 'did not take test'.

Includes a modal to check that the user is happy to proceed even if some applications are missing scores.

Enables user to proceed to the merit list and setting a pass mark.

Closes #2058 

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
For an exercise that has run a scenario test and has scores to be entered on platform, navigate to Tasks > Scenario test:

- When entering data for a scenario test leave some applications without scores.
- Press the Continue button and check that a warning modal is displayed.
- Confirm you'd like to continue even though some applications don't have scores.
- Check that you successfully proceed to the merit list screen (where pass mark and overrides can be configured)

Data entry screen:
<img width="1215" alt="image" src="https://github.com/jac-uk/admin/assets/8524401/2881763d-5c80-451e-86a2-8e85dc43789a">

Expected warning modal:
<img width="727" alt="image" src="https://github.com/jac-uk/admin/assets/8524401/d4fd0b9a-c4aa-43f3-bdf6-d68eb055dccd">

Expected destination:
<img width="1244" alt="image" src="https://github.com/jac-uk/admin/assets/8524401/d1a23692-a9b8-48ca-a169-0782588699b8">


## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
